### PR TITLE
Add error message for `unique_items` parameter in `conlist`

### PR DESCRIPTION
**Pull request by Kludex**
Selected Reviewer: @dmontagu

### CODE REVIEW

File: pydantic/fields.py [hunk 1]
```diff
@@ -593,7 +593,7 @@ def Field(  # noqa C901
         warn('`max_items` is deprecated and will be removed. use `max_length` instead', DeprecationWarning)
         if max_length is None:
             max_length = max_items
-    if unique_items:
+    if unique_items is not None:
```
Comment (Kludex) [inline pydantic/fields.py:596]: If it's `False`, we still want to warn the user.

File: pydantic/types.py [hunk 2]
```diff
@@ -230,8 +231,31 @@ def confrozenset(
...
+
+    Args:
+        item_type (type[AnyItemType]): The type of the items in the list.
+        min_length (int | None, optional): The minimum length of the list. Defaults to None.
+        max_length (int | None, optional): The maximum length of the list. Defaults to None.
+        unique_items (bool | None, optional): Whether the items in the list must be unique. Defaults to None.
+
+    Returns:
+        type[list[AnyItemType]]: The wrapped list type.
+    """
```
Comment (Kludex) [inline pydantic/types.py:250]: Do you prefer all docstrings in the same PR, or can I take the opportunity here?

**Comment by Kludex** [comment]
please review

**Comment by hramezani** [comment]
LTGM 👍

**Comment by Kludex** [comment]
> LTGM 👍

Can I merge it? 👀

**Review by hramezani — APPROVED** [review]

**Comment by hramezani** [comment]
> > LTGM 👍
> 
> Can I merge it? 👀

Yes

**Pull request merged**